### DISCUSSION TITLE
API /chart/{pk}/cache_screenshot/

### DISCUSSION THREAD

**Discussion by davidfabri** [apache/superset#18281] [Q&A / Help]
I'm unable to call a screenshot using the Chart REST API screenshot endpoint. 

### Expected results

response API 

### Actual results
Postman give this message:

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try
	again.</p>

#### How to reproduce the bug
to check API I use Postman
1. GET http://localhost:8088/api/v1/chart/142/cache_screenshot/
2. insert the authorization token 
3. send GET request

### Environment

Ubuntu 20.04
docker version
Client:
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.13.8
 Git commit:        afacb8b7f0
 Built:             Fri Dec 18 12:15:19 2020
 OS/Arch:           linux/amd64
 Experimental:      false
Server:
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.8
  Git commit:       afacb8b7f0
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.3.3-0ubuntu2.2
 runc:
  Version:          spec: 1.0.1-dev
 docker-init:
  Version:          0.18.0

- superset version: About Version: 0.999.0dev
- python version: `python 3.7.9`
- node.js version: `node -v`

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by kaavee315** [comment]
Is this fixed?

**Comment by davidfabri** [comment]
I fixed it, the problem was "gekodriver", now I use Chrome.

**Reply by amartincolville** [reply]
hi @davidfabri 
I am using chrome but still having the same issue. Could you please share maybe how this works for you? Thank you!
